src/config.rs

```diff
@@ -906,6 +906,7 @@ pub struct DbConfig {
     pub info_log_keep_log_file_num: u64,
     #[config(skip)]
     pub info_log_dir: String,
+    #[serde(with = "tikv_util::config::readable_size_in_decimal_base_serde")]
     pub rate_bytes_per_sec: ReadableSize,
```

Comment:
Why it has to be 10 based?

Reply:
It's a common practice to measure data transfer rate with 10 base, e.g. the data rate unit in grafana uses (and can only uses) `MB/s, GB/s`.

Reply:
But the EBS's throughput is calculated using MiB https://aws.amazon.com/ebs/features/
> ***Volume throughput is calculated as MB = 1024^2 bytes

Azure also uses the same unit https://docs.microsoft.com/en-us/azure/virtual-machines/disks-types
> where MBps = 10^6 Bytes per second

Grafana can use 2 based unit too as long as "Data (ICE) / bytes" is selected.

Reply:
GCE doesn't explicitly specify what unit it's using, but from calculation in the page https://cloud.google.com/compute/docs/disks/performance, it should also be MiB.
> 0.25 GB per second = 250 MB per second => 250 MB per second / 3.35

Reply:
Thanks for bringing up the info, but it seems both Azure and GCP uses 10 base unit? And for AWS, I think the footnotes are used to clarify special cases (they only use MiB for Max Throughput/Volume), other MBs are by default interpreted as 10 base.

Reply:
> A case in point would be the pricing page

If you are saying the page https://aws.amazon.com/ebs/pricing/, there is also a note in the page:
> *on this page GB = 1024^3 bytes

"GB" or "MB/s" doesn't necessary mean it's 10 based.

Reply:
Oh I didn't see that... But it doesn't change two facts: 1) MB/GB is by default (maybe even by legal standard) interpreted as 10 base (we wouldn't need footnotes for standard interpretation) 2) Data transfer rate (which is priced in MB/s) is 10 based.

Reply:
Can you show references about two "facts"? MB/GB can have different calculations in different context, I don't think there is a "default".

Reply:
I think right in this context (AWS/GCP/Azure), when document writers use `MB/GB` without footnotes, their underlying meaning is the default meaning. Or Wikipedia?

Reply:
And for the (2) fact, I think it's self-proving to assume they interpret `MB/s` as `10e6 Bytes/s`, since they don't use footnotes to specify otherwise. But I guess I can't really prove that unless consulting their tech support.

Reply:
All your claim is based on the assumption "MB/GB is by default (maybe even by legal standard) interpreted as 10 base". However, even the page you quoted also mention that:
> By the end of 2009, the IEC Standard had been adopted by the IEEE, EU, ISO and NIST. Nevertheless, the term megabyte continues to be widely used with different meanings:
>
> Base 10
> ...
> Base 2
> ...
> Mixed
> ...

Reply:
Seems AWS and Azure use different standards😢. But I agree with Jay, there is no default meaning for MB/s. Both are correct. It depends on the context.

Reply:
@BusyJay I don't think there is any ambiguity in a well-maintained, official, document's usage of `MB/GB`, if that's the case why don't we see AWS/GCP spelling that out in their documents? I admit in casual cases there can be mixed usage, that's why I didn't change the legacy implementation for interpreting `MB/GB`.

and @Connor1996 AWS and Azure both use 10 base for data rate, and AWS uses 2 base for *some* usage of `MB`, and all such usages are well notified.

Let's not forget the purpose of this PR, I'm trying to make tikv's configuration of disk IO rate in alignment with standard (more standard if you must insist) way of measuring. The most pressing issue is Grafana default uses `MB/s aka 10e6 B/s` as unit, even for Data(ICE) proposed here, the display unit is `MiB/s`. And even if we decide to change all occurrences of `Data Rate` to `Data (IDE) Rate` in grafana, we won't be able to change the cloud platforms' own cloud monitoring measuring.

Reply:
To add to the argument, I think data rate brings with itself the context to use `MB` as `1e6B`, as shown in [Data rate units](https://en.wikipedia.org/wiki/Data-rate_units), and any devices/services with a BW spec.

Reply:
I object to interpret MB/GB and other units as 10 based for following reasons:
1. The unit itself is ambiguous, parse the same value with different rules based on which field is using will make things worse.
2. We have already parse MB/GB as 2 based in the past, there is no good reason to break the compatibility.

I think there can be two ways to solve the align problems:
1. Do nothing. It's quite easy to input 150MBps as 150,000,000Bps.
2. Accept other form like 150MBps as 150e6Bps. "150e6" is already accepted by Rust parse, all we need to do is also accept "e" as part of numbers.

Reply:
Discussed offline, going to use a separate data rate unit to avoid confusion.